### PR TITLE
wakuv2-test.yml: adding ivan's public key

### PR DESCRIPTION
**Pull request by Ivansete-status**

### CODE REVIEW

**Review by jakubgs — APPROVED** [review]
No, I don't think you will be getting admin.

Is this key at least password protected? Preferably it would be on a YubiKey.

**Comment by Ivansete-status** [comment]
Hi @jakubgs , the key isn't protected yet. I ordered a YubiKey but it didn't arrive yet.

**Comment by jakubgs** [comment]
Then please generate an `ed25519` key that is password protected:
```
ssh-keygen -o -a 100 -t ed25519 -f ~/.ssh/id_ed25519 -C "joe@example.com"
```

File: ansible/group_vars/wakuv2-test.yml [hunk 1]
```diff
@@ -6,6 +6,7 @@ bootstrap__active_extra_users:
...
   - { name: daniel,  uid: 8005, groups: ['docker', 'dockremap'], key: 'ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIOPJlQ9iYVD2SNuW2ob2nvNHs+/O9PbcPc+dIepHNHO0 danielkaiser@status.im' }
+  - { name: ivan,    uid: 8006, groups: ['docker', 'dockremap'], key: 'ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABgQDtlrxIdNe2YU3dquXLN6e6gs2EYJx97nzui1jcBCYvRwpXTAzXgo2FuiISj8Ikx7dWJWGzKF0OGtwkzBLYuFEKPZhB55zCe2P76LS0ivObWnnGEuQvvaU7xx3arQt5Pm86PQTqAFw15aAm6/UO8kzwMlmSWiQCHaRqnaCnaO+jdI6aOxuFnvYVern3mijHsjY3BEWp7eb4lm2G3QrtNiZIJkeJHlZPk12CywtUKZJJ38TfwZAeubXtypra12+36y9+XXf86jjYmhsAy/4eOmkrU083tgFnyWwz+5yOHeOjYSbPgf+hUXWHIHXJpotRCjo53BT8CF/+oeo9Zt+S2g0GGdxWFhN9gGp+kVYBzdhw1nA5ozvXcJEnysuv/Fzy9MlyShIxMxzmEslfJxI/jqWok5mB56uom38YAbcAt50STqwGz3aloCH49zva8GY0Il6AhWb27hUc/kxUkSpPbH4Zt2RqCqI7v9ql0c2ftYqhJ5gc2thxurp3eiZcDyLLbl8= ivanchi@lenovo' }
```
Comment (Ivansete-status) [inline ansible/group_vars/wakuv2-test.yml:9]:
```suggestion
  - { name: ivan,    uid: 8006, groups: ['docker', 'dockremap'], key: 'ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIJBdm8y1PfWjT1pioaWJSZ2ETrUySb+dS/ifDg+VIpLY ivansete@status.im' }
```

Comment (jakubgs) [reply ansible/group_vars/wakuv2-test.yml:9]: Please update the PR then.

Comment (Ivansete-status) [reply ansible/group_vars/wakuv2-test.yml]: I think not it looks much better. Thanks!

**Comment by Ivansete-status** [comment]
That won't happen again. Apologies.

**Comment by jakubgs** [comment]
Cool. Deployed, you should have access to the test fleet now.

**Pull request merged**